tests/serverDescription/serverDescription-var_export-001.phpt

```diff
@@ -0,0 +1,31 @@
+--TEST--
+MongoDB\Driver\ServerDescription: var_export()
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_not_live(); ?>
+<?php skip_if_not_clean(); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$manager = create_test_manager();
+$server = $manager->selectServer(new MongoDB\Driver\ReadPreference('primary'));
+
+echo var_export($server->getServerDescription(), true), "\n";
+
+?>
+===DONE===
+<?php exit(0); ?>
+--EXPECTF--
+MongoDB\Driver\ServerDescription::__set_state(array(
+   'host' => '%s',
+   'port' => %d,
+   'type' => %d,
+   'hello_response' => 
+  array (
+    %a
+  ),
```

Comment:
It's very bizarre that `var_export` uses a three-space indent for keys, but a two-space indent for this `array (` output. I can confirm that's the case locally for PHP 7.4, though, so this is fine.